app-benchmarks/stress/Manifest

```diff
@@ -1 +1,2 @@
 DIST stress-1.0.4-r1.tar.gz 191800 BLAKE2B 560065d963823e734bb706840908998a58c07c82b21ab579f7a0fd5163e7a8591ca0f6abe430bba058bafd9a2dac1fcf33f011663cd6c3d2fdefb221a49e5e8f SHA512 623a7d9585828bed4b7673e19fb0cfdda904552cd56353d74106047b64249299d5f8da892ec135f1781ec3984c7d5835dcc6c0d8e659e06c8a1e4fb2af9fd9c4
+DIST stress-1.0.4.tar.gz 191800 BLAKE2B 560065d963823e734bb706840908998a58c07c82b21ab579f7a0fd5163e7a8591ca0f6abe430bba058bafd9a2dac1fcf33f011663cd6c3d2fdefb221a49e5e8f SHA512 623a7d9585828bed4b7673e19fb0cfdda904552cd56353d74106047b64249299d5f8da892ec135f1781ec3984c7d5835dcc6c0d8e659e06c8a1e4fb2af9fd9c4
```

Comment:
Why are you renaming the distfile (again)?

Reply:
Because the current version renames the original distfile to `-r1`, which doesn't make sense IMHO.

```
SRC_URI="https://people.seas.harvard.edu/~apw/stress/${MY_P}.tar.gz -> ${MY_P}-r1.tar.gz"
```

 So I did choose in purpose to drop the `-r1` for the distfile. I also plan to drop the old version, when `-r2` will be stable.

Reply:
It did because the distfile has changed. There's no point in forcing users to refetch again for no reason.

Reply:
Ok, changed.